shell/assets/translations/zh-hans.yaml

```diff
@@ -4986,47 +4996,80 @@ storageClass:
     warning: '{provisioner} 树内插件已弃用。你可以在<a target="_blank" rel="noopener noreferrer nofollow" href="https://kubernetes-csi.github.io/docs/drivers.html">这里</a>查找 CSI 驱动。'
 
 tableHeaders:
+  assuredConcurrencyShares: 确保并发份额
   accessKey: Access Key
+  addressType: 地址类型
+  accessModes: 访问模式
   address: 地址
   age: 存活时间
   apiGroup: API 组
+  apikey: API 密钥
+  available: 可用
+  attachedVM: 挂载的虚拟机
+
   authRoles:
     globalDefault: 新用户的默认角色
     clusterDefault: 集群创建者的默认角色
     projectDefault: 项目创建者的默认角色
   branch: 分支
+  backupTarget: 备份路径
   builtIn: 内置
   bundlesReady: 就绪 Bundle
   bundleDeploymentsReady: Deployment
   builtin: 内置
   chart: Chart
+  capacity: 容量
+  current: 当前
   clusterCreatorDefault: 集群创建者的默认角色
   clusterFlow: Cluster Flow
   clusterOutput: 集群输出
   cluster: 集群
   clusters: 集群
+  cluster-Name: 集群
   clustersReady: 就绪集群
   clusterGroups: 集群组
   commit: Commit
   condition: 状态
+  completions: 完成 Job 历史数
+  count: 数量
+  createdAt: 创建于 
   customVerbs: 自定义操作
   description: 描述
-  duration: 持续时间
   expires: 过期时间
-  providers: 提供商
   cpu: CPU
+  currentReplicas: 当前副本数
   date: 日期
+  data: 数据
   default: 默认
+  desired: 期望
+  defaultVersion: 默认版本
   destination: 目标
+  deployed: 已部署
   download: 下载
+  duration: 持续时间
+  diskState: 磁盘状态
+  drivers: 驱动
+  distinguisherMethod: 区分方法
   effect: 效果
   endpoints: 端点
+  firstSeen: 首次出现时间
   fleetBundleType: 类型
   flow: Flow
+  fingerprint: 唯一标识
   gitRepos: Git 仓库
+  groups: 组
+  groupName: 组名称
+  groupRoleNames: 组角色名称
+  global-Default: 全局默认
   host: 主机
+  hostIp: 主机 IP
+  holder: 持有者
+  hpaReference: 工作负载
   health: 健康
+  handSize: Hand Size
+  ipAddress: IP 地址
   id: ID
+  iP: IP
```

Comment:
Hi @bisht-richa, Similar to `Nodename` above, it seems strange to use `iP` as the primary key, we usually use `ip` or `IP`

Reply:
These are used in sortable tableHeaders.